initial-promises/node-server/common/1.0/site.cf

```diff
@@ -44,78 +44,95 @@ bundle common g
+      "excludedreps"                slist  => { "\.X11", ".*kde.*", "\.svn", "perl" };
+      "ncf_local_excludedreps"      slist => { "50_techniques", @{g.excludedreps}};
+      "rudder_tools_origin"         string => "/var/rudder/tools";
+
+      # Path where all the configurations (git with techniques, groups, ncf, etc) are stored
+      "rudder_configuration_repository"
+                                    string => "";
+
+      "rudder_ncf_origin_common"    string => "/usr/share/ncf/tree";
+      "rudder_ncf_origin_local"     string => "${rudder_configuration_repository}/ncf";
+
+      "rudder_ncf_hash_file"          string => "ncf_hash_file";
+      "rudder_ncf_origin_common_hash" string => "${rudder_ncf_origin_common}/${rudder_ncf_hash_file}";
+      "rudder_ncf_origin_local_hash"  string => "${rudder_ncf_origin_local}/${rudder_ncf_hash_file}";
+
+      "uncompressed_inventory_file_types" slist => { ".*.ocs" };
+      "inventory_file_types"              slist => { "@{uncompressed_inventory_file_types}", ".*.gz" };
+      "signed_inventory_file_types"       slist => { "@{inventory_file_types}", ".*\.sign" };
+
+      "server_roles_path"           string => "${rudder_base}/etc/server-roles.d/";
+
+
+      "rudder_tools_updated_filename" string => "rudder_tools_updated";
+      "rudder_tools_updated_origin" string => "${rudder_tools_origin}/${rudder_tools_updated_filename}";
+      "rudder_tools_updated"        string => "${rudder_tools}/${rudder_tools_updated_filename}";
 
     # The time at which the execution started
     windows.!cygwin::
-      "execRun" string => execresult("c:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command \"'{0:yyyy-MM-dd HH:mm:sszzz}' -f (get-date)\"", "noshell");
+      "execRun"                    string => execresult("c:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command \"'{0:yyyy-MM-dd HH:mm:sszzz}' -f (get-date)\"", "noshell");
     android::
-      "execRun" string => execresult("/system/xbin/date \"+%Y-%m-%d %T%z\" | sed 's/\([-+][0-9][0-9]\)\([0-9][0-9]\)$/\1:\2/'", "useshell");
+      "execRun"                    string => execresult("/system/xbin/date \"+%Y-%m-%d %T%z\" | sed 's/\([-+][0-9][0-9]\)\([0-9][0-9]\)$/\1:\2/'", "useshell");
     (!windows|cygwin).!android::
       # We would like to use date's "--rfc-3339=second" option here, but it is not available on older OSes (RHEL 3/4, AIX 5...)
-      "execRun" string => execresult("/bin/date -u \"+%Y-%m-%d %T+00:00\"", "noshell");
+      "execRun"                    string => execresult("/bin/date -u \"+%Y-%m-%d %T+00:00\"", "noshell");
+
+      # Heartbeat interval, in minutes. Only used when in "changes-only" compliance mode
+      # In this mode, we should only send a "StartRun"/"EndRun" message pair if changes occur,
+      # or if this interval has elapsed since we last sent one
+      "heartbeat_interval" string => "";
+
+      "rudder_node_config_id" string => "";
```

Comment:
These 2 are in the same case as davuser and davpw, they are in variables.json but for some reason have not appeared in this generated file.